docs/extensions/record-video.md

```diff
@@ -0,0 +1,90 @@
+# record-video
+
+This extension records video from the participant's webcam during a trial. 
+
+This extension encodes the video data in [base 64 format](https://developer.mozilla.org/en-US/docs/Glossary/Base64). 
+This is a text-based representation of the video which can be coverted to various video formats using a variety of [online tools](https://www.google.com/search?q=base64+video+decoder) as well as in languages like python and R.
+
+## Parameters
+
+### Initialization Parameters
+
+Initialization parameters can be set when calling `initJsPsych()`
```

Comment:
This is a little confusing. Can we say something here like "This extension has no initialization parameters." and/or put _None_ under the table headings below?